src/main/java/com/udemy/videolist/service/VideoService.java

```diff
@@ -0,0 +1,19 @@
+package com.udemy.videolist.service;
+
+import com.udemy.videolist.model.Video;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface VideoService {
+
+    List<Video> findAllVideos();
+
+    Video findByVideoId(int id);
+
+    List<Video> findByLanguage(String language);
+
+    List<Video> findByPrice(String price);

```

Comment:
金額なら整数（int）が適切かと思いました。
```suggestion
    List<Video> findByPrice(int price);

```